### ISSUE TITLE
Edit linked list in Set up DNS records for Cloud Office email and Skype for Business article

### ISSUE THREAD

**Issue by kylelaffoon** [rackerlabs/rackspace-how-to#1048]
Renate Burns submitted linking issue as follows:
When I input this:
 
-   [MX records](#### **MX records**)
-   [Autodiscover record](#### **Autodiscover records**)
-   [SPF record](#### **SPF records**)
 
Here’s what the preview shows me (screenshot):

 
Is this correct?
 
Thanks,
Renate
 
From: Kyle Laffoon 
Sent: Tuesday, June 7, 2016 3:46 PM
To: Renate Burns <Renate.Burns@rackspace.com>
Subject: Re: Question about editing How To articles
 
Looks like we were "fancy" on that heading with the recent migration. For your link, please use [Autodiscover record](#### **Autodiscover records**) .
 
Thanks,
Kyle
 
From: Renate Burns <Renate.Burns@rackspace.com>
Date: Tuesday, June 7, 2016 at 3:21 PM
To: Rackspace Hosting <kyle.laffoon@RACKSPACE.COM>
Subject: RE: Question about editing How To articles
 
Sure, it’s this one: https://support.rackspace.com/how-to/set-up-dns-records-for-cloud-office-email-and-skype-for-business/
 
Renate
 
From: Kyle Laffoon 
Sent: Tuesday, June 7, 2016 3:13 PM
To: Renate Burns <Renate.Burns@rackspace.com>
Subject: Re: Question about editing How To articles
 
That should be correct. Can you send the url for the article being edited? I can jump in there to verify I'm setting you up with the correct heading level. We typically use ### (H3) or #### (H4).
 
-Kyle
 
From: Renate Burns <Renate.Burns@rackspace.com>
Date: Tuesday, June 7, 2016 at 2:41 PM
To: Rackspace Hosting <kyle.laffoon@RACKSPACE.COM>
Subject: RE: Question about editing How To articles
 
Hi Kyle,
Thanks for getting back to me so quickly! Yes, I am wanting to link to an anchored area on the same page. So then, if the page heading is Autodiscover, for the navigation link I would put [Autodiscover](###Autodiscover) ? Is it case sensitive?
 
Thanks again!
 
Renate
 
From: Kyle Laffoon 
Sent: Tuesday, June 7, 2016 12:53 PM
To: Renate Burns <Renate.Burns@rackspace.com>
Subject: Re: Question about editing How To articles
 
Hi Renate,
 
I am the correct person to talk to about this.  In markdown, you're entering the text to appear for the link in brackets [ ], followed by the url in parentheses ( ). 
 
Are you attempting to link to an anchored area of the same page? If so, try linking to the heading immediately preceding the area [linkedHeading](###heading). Notice that you will include the # tags along with the heading reference in parentheses.
 
Let me know if that's still unclear.
 
-Kyle
 
 
From: Renate Burns <Renate.Burns@rackspace.com>
Date: Monday, June 6, 2016 at 4:36 PM
To: Rackspace Hosting <kyle.laffoon@RACKSPACE.COM>
Subject: Question about editing How To articles
 
Hello!
 
I recently asked one of my team leads about editing one of Cloud Office’s how to articles, and was told to go ahead and do it. I’m not familiar with the markup used, and was referred to you to check if what I’m thinking is correct.
 
Basically, I want to  create anchor links in one of our articles. Some Googling got me here: http://stackoverflow.com/questions/6695439/how-do-you-create-link-to-a-named-anchor-in-multimarkdown (top answer) - but I’m not sure if this is how to do it.
 
Can you enlighten me as to how anchor links work in this system?
 
Thank you! J
 
Renate Burns
Customer Service Technician I
 
rackspace_logo
 
cid:6A3CFAB6-9D63-4677-9494-0B1FFBA09E65

**Comment by kylelaffoon** [comment]
Sent Renate the updated format for her list and opened PR 1049 with change implemented. Asking Renate if she prefers to submit the change (for practice). I will submit or close my PR based on her choice.
